code/modules/projectiles/ammo_datums.dm

```diff
@@ -429,14 +429,22 @@
 
 /datum/ammo/bullet/pistol/heavy/super/highimpact
 	name = ".50 high-impact pistol bullet"
-	penetration = ARMOR_PENETRATION_TIER_2
-	debilitate = list(0,2,0,0,0,1,0,0)
+	penetration = ARMOR_PENETRATION_TIER_1
+	debilitate = list(0,1.5,0,0,0,1,0,0)
 	flags_ammo_behavior = AMMO_BALLISTIC
 
+/datum/ammo/bullet/pistol/heavy/super/highimpact/ap
+	name = ".50 impact armor piercing pistol bullet"
+	penetration = ARMOR_PENETRATION_TIER_10
+	damage = 45
+
 /datum/ammo/bullet/pistol/heavy/super/highimpact/New()
 	..()
 	RegisterSignal(src, COMSIG_AMMO_POINT_BLANK, .proc/handle_battlefield_execution)
 
+/datum/ammo/bullet/pistol/heavy/super/highimpact/ap/New()
```

Comment:
unnecessary - subtypes already utilize their parents' procs if not overriden. remove this line as New() is called with your intended stats above.